### ISSUE TITLE
Restrict the abnormalLevelOfChemicalEntity pattern to anatomical entities

### ISSUE THREAD

**Issue by rays22** [obophenotype/upheno#981]
Restrict the abnormalLevelOfChemicalEntity pattern to anatomical entities

- [ ] Update the EQ in `src/patterns/dosdp-patterns/abnormalLevelOfChemicalEntity.yaml`

so that the attribute is restricted chemicals in anatomical entities:

```
'has part' some 'amount'
        and (('characteristic of' some ('%'
            and ('part of' some 'anatomical entity')))
            and ('has modifier' some 'abnormal'))
% = chemical entity
```

This issue is to follow up https://github.com/obophenotype/bio-attribute-ontology/pull/367#pullrequestreview-2759379587 .

**Comment by sbello** [comment]
What does this mean for abnormal level in exhaled gasses? See for example http://purl.obolibrary.org/obo/HP_0033036 while it is called nasal the thing being measured is the level of nitric oxide in the exhaled gas. It is only 'nasal' because that is where the probe is.

HPO also has
[Positive hydrogen breath test](https://www.informatics.jax.org/vocab/hp_ontology/HP:6000259)
[Positive lactose hydrogen breath test](https://www.informatics.jax.org/vocab/hp_ontology/HP:4000100) 

Also we already have a pattern for chemical level in location https://raw.githubusercontent.com/obophenotype/upheno/master/src/patterns/dosdp-patterns/abnormalLevelOfChemicalEntityInLocation.yaml 
This change would make the two patterns identical.

I'm inclined we just obsolete the pattern without location. The HPO terms do not have EQs now and Positive hydrogen breath test and Positive lactose hydrogen breath test both refer to elevated hydrogen in the exhaled breath with the only difference being what the person consumed before the test.

**Comment by matentzn** [comment]
> I'm inclined we just obsolete the pattern without location. 

This will make naming (what is the standard label etc) and the transition for this pattern much harder. the addiotion @rays22 suggests really does not create _any_ work for any ontology using patterns (so, every ontology other than HPO and FYPO), while obsoleting this pattern will create a big amount of work for all patternised POs..

Your question about http://purl.obolibrary.org/obo/HP_0033036 is good though.. This is something @rays22 needs to think through :P

**Comment by sbello** [comment]
Agreed about the obsolete comment. This was written before you pointed out that the difference between this pattern and the inLocation pattern is that the location in this pattern is static not variable.  I do think this needs a note at the top of the pattern to make it super clear. 

And yes, what do we do about http://purl.obolibrary.org/obo/HP_0033036 or similar terms. Although, I do see that Uberon has a term for bodily gas http://purl.obolibrary.org/obo/UBERON_0034873 that covers all expelled gasses so we could ask for a term to cover exhaled gas. There is already one for excreted gas.
